Makefile

```diff
@@ -58,15 +58,24 @@ import-centos7:
 	make weld-f25
 	make importer
 	mkdir rpms/
-	sqlite3 centos-metadata.db < schema.sql
 	pip install -r tests/requirements.txt
+
+	if [ -n "$$EXISTING_MDDB" ]; then             \
+	    wget "$$EXISTING_MDDB";                   \
+	    gunzip -q `basename "$$EXISTING_MDDB"`;   \
+	    if [ ! -e ${d}/mddb ]; then               \
+	        mkdir ${d}/mddb;                      \
+	    fi;                                       \
+	    UNZIPPED=`basename "$$EXISTING_MDDB" | sed 's/.gz//'`; \
+	    mv $$UNZIPPED ./mddb/$(MDDB);             \
+	fi
+
```

Comment:
I've had this initially but `entrypoint.sh` creates the DB for us, if it doesn't exist, so no need for duplication.